app.js

```diff
@@ -12,7 +12,6 @@ const close = () => {
 };
 openMenu.addEventListener('click', show);
 closeMenu.addEventListener('click', close);
-
 document.querySelectorAll('.menu').forEach((menu) => {
   menu.addEventListener('click', () => {
```

Comment:
- [ ] Currently, when the user clicks on a navbar link on the desktop view, it goes to the correct portion of the page and closes the menu, which is the right behavior for this feature 👏🏻. However, if you then scroll up, you will see that the navbar links disappeared. Kindly fix this bug.
![image](https://user-images.githubusercontent.com/62290545/168615177-62ed905c-cbc6-4aa2-8a51-6427f11c089c.png)


- [ ] When the mobile menu is closed, the close menu icon is still displayed on mobile and desktop views. Please fix this bug.
   - On mobile
![image](https://user-images.githubusercontent.com/62290545/168618054-3c68312e-fedc-40c3-ad83-e95a1f56424e.png)
   - On desktop
![image](https://user-images.githubusercontent.com/62290545/168618199-9a3babd1-f30e-4852-88f7-dd32df021520.png)